bestbook.egg-info/requires.txt

```diff
@@ -0,0 +1,3 @@
+virtualenv>=1.11.6
```

Comment:
Why not just use the venv module that is built into the Python standard library?

Reply:
This may be a server-specific configuration?  It may be worthwhile to update [setup.py](https://github.com/Open-Book-Genome-Project/TheBestBookOn.com/blob/master/setup.py) when we begin using Docker for this app.